### DISCUSSION TITLE
Feedback from a Beginner

### DISCUSSION THREAD

**Discussion by Manos1966** [claytonjn/hass-circadian_lighting#250] [General]
Hallo,
I just want to give you my Feedback as a new user, after trying to install the project:

 - Installation Instructions
 First part was easily understandable, from the three chjoices I had, I used the simplest (HACS)
The next step was more complicated:
NOTE: Custom Updater will install the files, but you still need to enable Circadian Lighting in your configuration by following the instructions [here](https://github.com/claytonjn/hass-circadian_lighting#basic-installationconfiguration-instructions).

- The "here" i.e. https://github.com/claytonjn/hass-circadian_lighting#basic-installationconfiguration-instructions
Is of limited use for a beginner in my opinion.

Just my 2 cents.

**Comment by Olsen93** [comment]
Hi there, 
the same here, almost 2 years later, though.
This App looks promising, but I have no idea how to get it to work.
